.github/workflows/ruby_test.yml

```
@@ -0,0 +1,21 @@
+name: CI
+
+on: pull_request
```

```suggestion
on:
  pull_request:
  push:
    branches:
      - 'master'
      - '*-stable'
```

That way we get a cache that can be re-used on next runs